usagov/usagov-2021
# USAGOV-1149-Line-Height: Footer, Breadcrumb and Scams Line Height

## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-1149

## Description

Updated line heights for footer, breadcrumbs and scams wizard

## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions



### Requires New Config

### Requires New Content

### Validation Steps
Test these elements please 
- Email sign up text ----flagging 1.3 should be 1.5 (is always a single line, but technically should be 1.5)
- Sign up button ---- flagging because 1.5, secondary button and toggle text are .9
- SM header Official guide banner copy----flagging 1.3 should be 1.5 but single line
- Breadcrumbs ---flagging 1.15 should be 1.5 for when it runs to 2 lines
- Scams tool radio buttons ---- flagging 1.3 should be 1.5

## Security Review




## Reviewer Reminders
- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions
Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
